src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RANDOM = new Random().nextInt(5);
```

Comment:
I don't think we need it as method parameter, let's move it to the method getRandomFigure()

Reply:
I rewrite it so:
int random = myRandom();|
in method getRandomFigure()